scripts/events.js

```diff
@@ -1,31 +1,36 @@
 'use strict';
 
-var proNavsAndFilters = { };
+(function(module) {
```

Comment:
Good job wrapping your modules in IIFEs to only expose an interface to the window through which you can access your methods!